roles/dtc/create/tasks/vrfs_networks.yml

```diff
@@ -56,3 +55,16 @@
     - MD.vxlan.overlay_services.networks is defined
     - MD.vxlan.overlay_services.networks
     - changes_detected_networks
+
+# --------------------------------------------------------------------
+# Manage Loopback VRF attachments on NDFC
+# --------------------------------------------------------------------
+- name: Attach VRF Loopbacks per VRF
+  include_tasks: vrf_loopbacks.yml
+  loop_control:
+    loop_var: vrf
+  loop: "{{ MD.vxlan.overlay_services.vrfs }}"
```

Comment:
Are we able to refactor this to not loop?

Reply:
removed, left over from an old looping version

Reply:
should be fixed now